### PR TITLE
Changed default SendMetricsInterval to 60s

### PR DESCRIPTION
**Pull request by andreas-unleash**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Changes the default SendMetricsInterval to 60s as per our [documentation](https://docs.getunleash.io/sdks#server-side-sdk-compatibility-table)

Fixes # https://github.com/Unleash/unleash-client-dotnet/issues/105

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by sighphyre — COMMENTED** [review]
This seems fine but what's the rationale behind this change?

**Comment by andreas-unleash** [comment]
There was a mismatch with our documentation - where we specify the default send metrics interval to be 60s for this client.

**Comment by sighphyre** [comment]
> There was a mismatch with our documentation - where we specify the default send metrics interval to be 60s for this client.

Gotcha, that makes sense. We should fix the relevant tests before we merge this though

File: src/Unleash/UnleashSettings.cs [hunk 1]
```diff
@@ -63,11 +63,11 @@ public class UnleashSettings
         public TimeSpan FetchTogglesInterval { get; set; } = TimeSpan.FromSeconds(30);
 
         /// <summary>
-        /// Gets or sets the interval in which metriccs are sent to the server. When null, no metrics is sent.
+        /// Gets or sets the interval in which metrics are sent to the server. When null, no metrics is sent.
```
Comment (sighphyre) [inline src/Unleash/UnleashSettings.cs:66]: Thanks for fixing the spelling here! Got a little grammar fix while we're here

```suggestion
        /// Gets or sets the interval in which metrics are sent to the server. When null, no metrics are sent.
```

File: build.cake [hunk 2]
```diff
@@ -13,6 +13,9 @@ var solutionPath = "./Unleash.sln";
 var unleashProjectFile = "./src/Unleash/Unleash.csproj";
 var buildDir = Directory("./src/Unleash/bin") + Directory(configuration);
 
+var csTestsVersion = "v4.1.0"
+var clientSpecificationTestsURL = string.Format("https://raw.githubusercontent.com/Unleash/client-specification/{0}/specifications/", csTestsVersion);
```
Comment (sighphyre) [inline build.cake:17]: I think this is breaking the build

```suggestion
var clientSpecificationTestsURL = String.Format("https://raw.githubusercontent.com/Unleash/client-specification/{0}/specifications/", csTestsVersion);
```

**Review by sighphyre — APPROVED** [review]
LGTM!

**Pull request merged**